méli-mélo/2022-09-svgimagemap/styles.css

```diff
@@ -0,0 +1,60 @@
+.mapdiv {
+	width: 33%;
+	margin: auto;
+}
+
+.mapdiv path {
+	stroke: #FFFFFF;
+	stroke-width: 5px;
+}
+
+.mapdiv polygon {
+	stroke: #FFFFFF;
+	stroke-width: 5px;
+}
+
+.st0 {
+	fill: #122E33;
+	stroke:#FFFFFF;
+	text-align:center;
+}
+
+.st1 {
+	fill: #99BDC0;
+	stroke:#FFFFFF;
+}
+
+.st2 {
+	fill: #337C81;
+	stroke:#FFFFFF;
+}
+
+.st3 {
+	fill: #075C62;
+	stroke:#FFFFFF;
+}
+
+.st4 {
+	fill: #F1F2F2;
+	stroke:#FFFFFF;
+}
+
+.st5 {
+	font-family: 'Arial-BoldMT';
+}
+
+.st6 {
+	font-size: 66.613px;
+}
+
+.st7 {
+	fill: #414042;
+}
+
+#bcregion a:hover .st0, #bcregion a:active .st0, #bcregion a:focus .st0, #bcregion a:hover .st1, #bcregion a:active .st1, #bcregion a:focus .st1, #bcregion a:hover .st2, #bcregion a:active .st2, #bcregion a:focus .st2, #bcregion a:hover .st3, #bcregion a:active .st3 , #bcregion a:focus .st3 {
```

Comment:
Can you replace the id selector by a CSS class selector. This is to ensure the code re-usability.